cmd/buildctl/build/exportcache.go

```diff
@@ -39,6 +39,9 @@ func parseExportCacheCSV(s string) (client.CacheOptionsEntry, error) {
 	if _, ok := ex.Attrs["mode"]; !ok {
 		ex.Attrs["mode"] = "min"
 	}
+	if ex.Type == "gha" {
+		return bindGithubCacheAttributes(ex)
```

Comment:
`loadGithubEnv`

Reply:
Done in https://github.com/moby/buildkit/pull/2707/commits/6ac606c9b135222934dda2b55e19ac9820c02b7c